markdown/epilogue.markdown

```diff
@@ -1,39 +1,151 @@
+improve. You'll also become more familiar with the GitHub contributing process
+and etiquette, which are highly valuable skills in today's job market.
+
+### Conferences
+
+In the same vein, we highly recommend attending a coding conference in this field.
+SciPy, held every year in Austin, is fantastic, and probably your best bet if you
+enjoyed this book. There's also a European version, EuroSciPy, which changes host
+city every two years. Finally, the more general PyCon conference happens in the
+US but also has offshoots around the world, such as PyCon-AU in Australia, which
+has a "Science and Data" miniconference the day before the main conference.
+
+In all these cases, *stay for the sprints* at the end of the conference. A coding
+sprint is an intense session of team coding, and it is a fantastic opportunity to
+learn the process of contributing to open source, regardless of your skill level.
+It is how one of your authors (Juan) started in their open source journey.
+
+## Beyond SciPy
+
+The SciPy library is written not just in Python, but also in highly optimized C
+and Fortran code that interfaces with Python. Together with NumPy, and related
+libraries, it tries to cover most use cases that come up in scientific data
+analysis, and provides very fast functions for these. Sometimes, however, a
+scientific problem doesn't match at all with what's already in SciPy, and a pure
+Python solution is too slow to be useful. What to do then?
+
+[High-Performance Python](http://shop.oreilly.com/product/0636920028963.do),
+by Micha Gorelick and Ian Ozsvald, covers what you need to know in these
+situations: how to find where you *really* need performance, and the options
+available to get that performance. We highly recommend it.
+
+Here, we want to briefly mention two of those options that are particularly
+relevant in the SciPy world.
+
+First, Cython is a variant of Python that can be compiled to C, but then be
+imported into Python. By providing some type annotations to Python variables,
+the compiled C code can end up being a hundred or even a thousand times faster
+than comparable Python code. Cython is now an industry standard and is used in
+NumPy, SciPy, and many related libraries (such as scikit-image) to provide fast
+algorithms in array-based code.  Kurt Smith has written the simply-titled
+[Cython](http://shop.oreilly.com/product/0636920033431.do) to teach you the
+fundamentals of this language.
+
+An often easier-to-use alternative to Cython is Numba, a just-in-time compiler
+(JIT) for array-based Python. In Numba code, you don't need to annotate types:
+Numba will infer them when a function is first called. Instead, you simply need
+to make sure that you only use basic types (integers, floats, etc), and arrays,
+rather than more complicated Python objects. In these cases, Numba can compile
+the Python code down to very efficient code and gain orders of magnitude speedups.
+
```

Comment:
I like the split. The first paragraph is about Numba specifically, while the second one is about the state of JITs in Python in general. And at any rate, it would become a very long paragraph.